Library/Homebrew/cmd/update.sh

```diff
@@ -249,6 +252,8 @@ pull() {
     pop_stash_message
   fi
 
+  [[ -n "$HOMEBREW_VERBOSE" ]] && echo
```

Comment:
I think you can omit this here and instead move the `echo` to the top of the `pull` function, before `Updating $DIR...` is output, which simplifies things a tiny bit and will also create a visual separation between the fetch phase and the rebase/merge phase.

But you could still add this line after the loop over all repositories in `homebrew-update` to create some visual separation between the rebase/merge phase and the output of `update-report`.